src/test/java/website/frontrow/util/CollisionHandlerTest.java

```diff
@@ -152,7 +156,7 @@ public void platformMotionDownAABBTest()
      * Test getNextPosition in a terrible box of terribleness.
      */
     @Test
-    public void getNextPositionTestRetricted()
+    public void getNextPositionTestRestricted()
```

Comment:
You preface the other tests methods with test<name>, shouldn't you also do that here?